examples/creatives/hover_nonlinear.js

```diff
@@ -0,0 +1,89 @@
+const Y_OFFSET_PERCENTAGE = 8;
+const X_OFFSET_PERCENTAGE = 8;
+const WIDTH_PERCENTAGE = 1.1;
+const HEIGHT_PERCENTAGE = 2;
+
+class HoverNonLinear extends BaseSimidCreative {
```

Comment:
Add a description to what this sample is and what it does.

Reply:
added